### ISSUE TITLE
Some items didn't return to inventory after breaking sign

### ISSUE THREAD

**Issue by ghost** [EssentialsX/Essentials#2541]
### Information

**Full output of `/ess version`**:

```
<!-- Run the command in your console, and copy and paste the whole output here -->
```
[17:45:30 INFO]: CONSOLE issued server command: /ess version
[17:45:30 INFO]: Server version: 1.14-R0.1-SNAPSHOT git-Paper-15 (MC: 1.14)
[17:45:30 INFO]: EssentialsX version: 2.16.1.170-mc114-b87
[17:45:30 INFO]: Vault version: 1.7.2-b${env.TRAVIS_BUILD_NUMBER}

**Server log**: https://gist.github.com/Jonasslv/57a75067864f534fa770bb7b6748549b
(the server log is a mess since mc 1.14 keep spamming chunk errors).
**EssentialsX config**:  https://gist.github.com/Jonasslv/87e36e825ebaaffa3892ae44a358407f

### Details

**Description**  
Some Items didnt return to inventory after breaking a trade sign, the most recent was a trade sign of blazerod.

**Steps to reproduce**  
Did a trade sign selling 32 blazerod for $10 and then I broke it.
The items didnt came back to inventory.

**Expected behavior**  
The items getting back to inventory.

**Screenshots**  
![image](https://user-images.githubusercontent.com/20801365/57642632-35354480-758e-11e9-83be-8813d7801e43.png)
The report from the server.

**Comment by ghost** [comment]
@md678685 I made a video reproducing the issue, actually it happens when you break the block that holds the Wall Sign

https://www.youtube.com/watch?v=UeUwzKKVrQo

**Comment by minksuki** [comment]
I'm able to reproduce this and am working on a fix

**Comment by mdcfe** [comment]
Could you test this on the [latest experimental build](https://ci.ender.zone/job/EssentialsX-Experimental)? I've fixed #2529, which appears to be related.

**Comment by mdcfe** [comment]
It seems as though this was fixed with #2529.

@Jonasslv Could you try updating to the [latest build](https://essentialsx.github.io)?

**Comment by mdcfe** [comment]
Closing due to lack of response, plus I can't replicate this on the latest builds.
